### PR TITLE
Ingestion config for 10302 and required fixes

### PR DESCRIPTION
**Pull request by uermel**
- First stab at ingestion config for 10302
- changes to tomoman_to_tsv.py concerning this dataset: gainref location, check for presence of defect file entry

- fix for 10301: location of some rawtlt files.

### CODE REVIEW

File: ingestion_tools/dataset_configs/10302.yaml [hunk 1]
```diff
@@ -117,14 +117,14 @@ tomograms:
...
-  run_regex: /\d{8}_.*$
+  run_glob: "chlamy_visual_proteomics/*"
+  run_regex: /(\d{8}_.*)$
   run_name_regex: (.*)
-  frames_glob: null # "chlamy_visual_proteomics/{run_name}/frames/*.eer"
-  frames_name_regex: null #(.*)
-  gain_glob: null # "chlamy_visual_proteomics/{frame_gain_reference}"
+  #frames_glob: null # "chlamy_visual_proteomics/{run_name}/frames/*.eer"
+  #frames_name_regex: null #(.*)
+  #gain_glob: null # "chlamy_visual_proteomics/{frame_gain_reference}"
```
Comment (uermel) [inline ingestion_tools/dataset_configs/10302.yaml:127]: @manasaV3 Currently, the test import fails due to the frames_glob missing. It also failed when I set it to `null`. What is the correct value here to skip frames?

Comment (uermel) [reply ingestion_tools/dataset_configs/10302.yaml]: Using an empty string worked.

File: ingestion_tools/dataset_configs/10302.yaml [hunk 2]
```diff
@@ -0,0 +1,140 @@
+dataset:
+  authors: &dataset_authors
+    - name: Ron Kelley,
+      primary_author_status: true
+    - name: Sagar Khavnekar,
```
Comment (manasaV3) [inline ingestion_tools/dataset_configs/10302.yaml:5]: Can we remove the commas here as well?

**Review by manasaV3 — APPROVED** [review]

File: ingestion_tools/dataset_configs/10302.yaml (hunk 2 above)
Comment (uermel) [reply ingestion_tools/dataset_configs/10302.yaml]: Done.

**Pull request merged**